server/test/unit/vitalsTest/checkButtonHeartbeatTest.js

```diff
@@ -3,13 +3,16 @@ const { expect, use } = require('chai')
 const { describe, it } = require('mocha')
 const sinon = require('sinon')
 const sinonChai = require('sinon-chai')
+const rewire = require('rewire')
 const { DateTime } = require('luxon')
 
 // In-house dependencies
 const { helpers, factories } = require('brave-alert-lib')
 const db = require('../../../db/db')
 const { buttonFactory, buttonsVitalFactory } = require('../../testingHelpers')
-const vitals = require('../../../vitals')
+require('../../mocks/tMock')
+
+const vitals = rewire('../../../vitals')
```

Comment:
Cool. I'm glad you figured out the `rewire`!

In this particular case, since the code is ours, you could also have exported the `sendNotifications` function from `vitals.js`. But I like that you used `rewire` because that way we didn't have to do that export!